### ISSUE TITLE
RFC: Zen of PySlackers

### ISSUE THREAD

**Issue by iMerica** [pyslackers/community#35]
Lets use this RFC to draft some general principles that we can rely upon when making decisions in our Slack group. In the spirit of the [Zen of Python](https://www.python.org/dev/peps/pep-0020/), I propose that we keep them general instead of rigid.

**Comment by DeannaWagner** [comment]
https://pyslackers.com/terms/

When considering the **Code of Conduct** from the terms, it seems that an extra bullet is needed under 
**Our Standards**
_Examples of unacceptable behavior by participants include:_

Something like: 
- Unless there is a channel for political or other divisive topics available, we discourage discussion that is of no value to the tech community or is designed to cause discord therein.

Furthermore, a _Consequences of unacceptable behavior_ section immediately following and contained within the CoC would be helpful.  The moderators of the Slack team could come up with the steps they take given unacceptable behavior, perhaps taking into consideration:
1) the overall value the community member provides (ie. is it a new member posting spam, or is it a long time, positive contributor who is posting something unacceptable, and 
2) the severity of the unacceptable behavior; if it is something illegal then perhaps banning outright would be called for, but in other cases some sort of warning system.

Integrity does not only mean honesty.  It means having consistent standards that apply to all members, regardless of their role in an institution.  The PySlackers community is large, vibrant and in need of clear guidelines so that members can find there way in our community clearly and positively.  Moreover, having clear guidelines makes the job of moderation much more efficient.

**Comment by ovv** [comment]
I agree that we should write somewhere the various escalation and consequences. What @DeannaWagner propose seems to be a very good starting point :+1: .

**Comment by kootenpv** [comment]
I am still proposing "channeling" divisive content into their own channel (or a catch-all #controversive).

I imagine people in this community not only come to talk about "topic X", but also come to the community to "talk about topic X, _with pythondevs_".

I propose to mention in the ToS that we do not encourage divisive content.

When someone does bring up topics such as politics, refer them to the channel for this topic. That itself makes it so that other people won't be exposed more than necessary to these topics.

No one will be lost (or feel hurt), and this should be the same if not less amount of moderation as strongly enforcing not discussing certain topics.

**Comment by DeannaWagner** [comment]
In the interest of helping this community, I would like to share my experience with what can only be described as the Toxic Python Patriarchy.  I truly hope that some good will come of it and that your slack team in particular will become a more welcoming place for more than the moderator class rulers.

Yesterday I witnessed the absolute shutdown of my on-topic discussion by a self-professed feminist moderator and another who has been micromanaging me lately.  I brought up a correlation that I thought would be interesting knowledge.  Mod 1 has to school me that correlation isn't causation.  Oh mighty leader, don't you think you could have approached the little lady with a less heavy hand?  How about asking if I had more information on that or comment on it in a more neutral way?  Then Mod 2 has to come in and tag team bully me down, and wow, coming from 2 moderators, I guess I had better stfu, especially considering I am one of the only active females out of 14k+ users.  Moderator 2 often gives very specific nutrition advice to individuals.  Even I, a ACSM certified personal trainer would never hand out such advice on a forum.  Hope Mod 2 has a license to practice nutrition in this person's jurisdiction.

You lost another long time user recently, a real gem to the python community.  Brilliant person.  FOR THE SAME REASON.  Power tactics by the ruling elite.  If I was going to hand out advice, I would tell the patriarchy to take some extra strength egodeflateX, watch or read Animal Farm and get over yourselves.  Try diversifying and lay low on policing speech unless it is in clear violation of your terms, else you risk becoming a small group of elites with no peasant class to rule.
